shared/src/e2e/driver.ts

```diff
@@ -300,6 +300,16 @@ export class Driver {
                 await this.page.waitForSelector(`.repository-node[data-e2e-repository='${slug}']`, { visible: true })
                 // Workaround for https://github.com/sourcegraph/sourcegraph/issues/5286
                 await this.page.goto(`${this.sourcegraphBaseUrl}/${slug}`)
+
+                if (slug.toLowerCase().includes('alwayscloningtest')) {
```

Comment:
Afaict this is a specific repo - if I change the org to something else it returns 404 for me. Or am I missing something?
```suggestion
                if (slug.toLowerCase() === 'sourcegraphtest/alwayscloningtest') {
```
https://sourcegraph.com/github.com/sourcegraphtest/AlwaysCloningTest